parlai/core/torch_agent.py

```diff
@@ -277,27 +283,45 @@ def __init__(
         self.history_strings = []
         self.history_raw_strings = []
         self.history_vecs = []
+        self.history_timesteps = []
```

Comment:
Could you add a comment about the relation between this and other history variables here?